group-coordinator/src/main/java/org/apache/kafka/coordinator/group/metrics/GroupCoordinatorMetrics.java

```diff
@@ -75,31 +74,23 @@ public class GroupCoordinatorMetrics extends CoordinatorMetrics implements AutoC
     public static final String GENERIC_GROUP_REBALANCES_SENSOR_NAME = "GenericGroupRebalances";
     public static final String CONSUMER_GROUP_REBALANCES_SENSOR_NAME = "ConsumerGroupRebalances";
 
+    private final MetricName genericGroupsCountMetricName;
```

Comment:
nit: Should it be called groupsCountMetricName?

Reply:
MetricName also contains the tags so i split it into genericGroupsCount and consumerGroupsCount